### DISCUSSION TITLE
Strange jump to end of page

### DISCUSSION THREAD

**Discussion by darkWolf-PR** [orestbida/cookieconsent#563] [Q&A]
Hi, I found some very strange acting of CC on one of the website( www.wrapstyle.com ) I´ve been implementing it lately. 

From time to time, page jumps directly to the bottom end on pageload and it seems, when I open Dev Tools, it is focused on this line:

`<div class="cm-wrapper cc--anim" tabindex="-1">`

It does not happen every time and I can´t find any consistent way to simulate it. It happens more often if I have opened Dev tools and reloading page (I´ve been working of load optimizations lately, so that was pretty annoying as it instantly iniciated all the lazy-loaded stuff through the pretty long homepage). Buf I got it happen few times when opening new tab and just loading the page.

It stops if I turn the CC off, so it seems to be connected with it.

It seems to be connected with Firefox tho, because, I´ve been only getting this in FF, did few tries with Opera or Edge with Chromium and it did not happen there. But could by only because I just use the FF most of the time, so the chances are much higher.

Any ideas, what can cause this?

**Comment by orestbida** [comment]
Mhm, this is weird indeed. I will look into it and see If I can find the issue!

**Reply by darkWolf-PR** [reply]
Still trying to find out, why is this happening, but no success. Just really seems to be tied to Firefox, because collegues who work mostly in Chrome did not experienced this behaviour, just like website owners, who mostly work with Macs and iPhones(and thats kind of good, because otherwise, they would be calling like every day about the problem and I have no idea, how to fix it...).

**Reply by darkWolf-PR** [reply]
I´ve recently added cc on another webpage ( [www.ekocistirna.cz](https://www.ekocistirna.cz/) ), this time onepage top-down scrolling, build on older version of BS(3) with html template bought on some market and it does the same thing in Firefox.

**Reply by mariusmateoc** [reply]
Hi, I have the same problem, and it happens only when `disablePageInteraction: true,`. I was able to fix it using the following:

```scss
/** Disable page scroll **/
html.disable--interaction {
  &.show--consent,
  &.show--consent body {
    height: 100vh !important;  // NOTE: Prevent page scroll
  }
}
```

**Reply by darkWolf-PR** [reply]
Hmm, sadly, I´m not using disable interaction option on any page. It does make sense as this literally cuts the page at the end of viewport and if u have disabled interaction, it can work. 

But client nor my studio colleagues want cookie panel with disabled interaction. And I´m not even sure if it´s actually legal, because the web page should be normally accessible without giving consent, which pretty much also means without interacting with the consent panel at all...

And it´s unusable solution without disabled interaction. It has bad consequences-sticky menu stops being sticky at the point where starting viewport(starting height that was set with 100vh) ends for example.

**Reply by orestbida** [reply]
@darkWolf-PR, I updated the v3 branch with a focus handling tweak. Can you check if the issues persists?

**Reply by darkWolf-PR** [reply]
@orestbida Hi, had literally no time lately, but I´ve tried updating script to rc17 today and checked it a bit and it seems ok, No jumping to the bottom.
